ldap/servers/slapd/back-ldbm/db-bdb/bdb_bdbreader_glue.c

```diff
@@ -0,0 +1,491 @@
+/** BEGIN COPYRIGHT BLOCK
+ * Copyright (C) 2024 Red Hat, Inc.
+ * All rights reserved.
+ *
+ * License: GPL (version 3 or any later version).
+ * See LICENSE for details.
+ * END COPYRIGHT BLOCK **/
+
+#include "bdb_layer.h"
+#include <robdb.h>
+
+#define DEBUG 1
```

Comment:
Shouldn't it be 0 by default

Reply:
should not even be defined at all (since we are using #ifdef)